beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -2072,3 +2085,26 @@ func (s *Server) GetDepositSnapshot(w http.ResponseWriter, r *http.Request) {
 		},
 	)
 }
+
+func (s *Server) broadcastBlobSidecars(ctx context.Context, sb *eth.GenericSignedBeaconBlock) error {
```

Comment:
is it worth using the geneic signed beacon block here? 
if you just pass the Blobs and kzgProofs from the argument it'll save the New SignedBeacon block and Get deneb calls ( not sure if that saves anything) but I think you have the data of denebBlock already here